site/styles/_shift72.scss

```diff
@@ -234,15 +234,6 @@
   @extend .form-check-input;
 }
 
-.s72-form-control-readonly,
```

Comment:
This style makes readonly fields appear as text which is different for the shopping readonly fields. E.g.

![image](https://user-images.githubusercontent.com/984586/227060901-d1469af5-be99-4ead-be7b-9d12a38de6e5.png)

We should probably have one way of styling readonly text inputs.